tardis/plasma/properties/atomic.py

```diff
@@ -39,7 +39,7 @@ def _set_index(self, levels):
         return (levels.index, levels['energy'], levels['metastable'],
             levels['g'])
 
-class Lines(BaseAtomicDataProperty):
```

Comment:
we should continue to use the `BaseAtomicDataProperty`. If there is a problem with the ordering then we need to change the `set_index` method.